skeleton/aws/main.tf

```diff
@@ -20,3 +20,19 @@ provider "aws" {
 }
 
 # VPC
+
+# Security groups
```

Comment:
I'm not sure we add these placeholders here, because, for the basic AWS template, we don't have it 🤔 for example the bastion instance, SSM